src/style.css

```diff
@@ -1 +1,53 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
```

Comment:
You should use a fallback font in case Avenir is not available. This could be a system font such as Arial or Helvetica. You can specify a list of fallback fonts in the `font-family` property. The browser will try to use the fonts in the order they are listed. If Avenir is not available, it will use the next font in the list.